### PR TITLE
update cyber security table

### PR DESCRIPTION
**Pull request by segunajibola**
check the cyber security table, week number can be added to others if its okay

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,29 +20,31 @@ Contents are added frequently! ⚡
...
-- [Mobile](#mobile)
-- [Productivity](#productivity)
-- [Programming](#programming)
-- [Project Management](#project-management)
-- [Science](#science)
-- [Software Engineering](#software-engineering)
-- [Startups](#startups)
-- [Thanks to all Contributors 💪](#thanks-to-all-contributors-)
-- [Thanks to all Stargazers ⭐️](#thanks-to-all-stargazers-️)
+- [Awesome Article Collection](#awesome-article-collection)
```
Comment (Olanetsoft) [inline README.md:23]: I think you have some MD extensions that auto-added additional bullet points here, kindly review them.

File: README.md [hunk 2]
```diff
@@ -20,29 +20,31 @@ Contents are added frequently! ⚡
...
+  - [Table of Contents](#table-of-contents)
```
Comment (Olanetsoft) [inline README.md:24]: I think you have some MD extensions that auto-added additional bullet points here; kindly review them.

File: README.md [hunk 3]
```diff
@@ -20,29 +20,31 @@ Contents are added frequently! ⚡
...
+  - [Machine Learning](#machine-learning)
+  - [Marketing](#marketing)
+  - [Mobile](#mobile)
+  - [Productivity](#productivity)
+  - [Programming](#programming)
+  - [Project Management](#project-management)
+  - [Science](#science)
+  - [Software Engineering](#software-engineering)
+  - [Startups](#startups)
+    - [Thanks to all Contributors 💪](#thanks-to-all-contributors-)
```
Comment (Olanetsoft) [inline README.md:46]: Readjust this too

File: README.md [hunk 4]
```diff
@@ -54,11 +56,9 @@ Contents are added frequently! ⚡
 ---
 
 ## Cyber Security
-| Article Title | Author | Article Link |
-| ------------- | ------ | ------------ |
-|               |        |              |
-
----
+| Week Number | Article Title | Author | Article Link |
```
Comment (Olanetsoft) [inline README.md:59]: I think we should have it monthly instead, or we should make it open. Remember, the goal is to encourage everyone, and now that we have a repo, we can accept more. What do you think? @segunajibola

**Review by Olanetsoft — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (segunajibola) [reply README.md:23]: Oops, that a Markdown extension I use, fixed that

File: README.md (hunk 3 above)
Comment (segunajibola) [reply README.md:46]: done

File: README.md (hunk 4 above)
Comment (segunajibola) [reply README.md:59]: I don't understand this. The week number will be for the week number the article was published. week 1 - 52.

Comment (segunajibola) [reply README.md:59]: Yeah, I'll do that now

**Comment by segunajibola** [comment]
Check now @Olanetsoft

**Pull request merged**